src/Serilog/Parsing/MessageTemplateParser.cs

```diff
@@ -43,7 +43,7 @@ public MessageTemplate Parse(string messageTemplate)
 
         static IEnumerable<MessageTemplateToken> Tokenize(string messageTemplate)
         {
-            if (messageTemplate == "")
+            if (messageTemplate.Length == 0)
```

Comment:
`.Length` is faster and we're guarded against null from the public `Parse` method